modules/_node-scripts/package.json

```diff
@@ -16,13 +16,16 @@
 		"prettier": "^3.2.5",
 		"resolve": "^1.19.0",
 		"stylelint": "13.2.0",
-		"typescript": "^5.4.5"
+		"tar": "^7.2.0",
+		"typescript": "^5.4.5",
+		"unzipper": "^0.11.6"
 	},
 	"exports": "./index.js",
 	"name": "@liferay/node-scripts",
 	"scripts": {
 		"checkFormat": "node-scripts format --check",
-		"format": "node-scripts format"
+		"format": "node-scripts format",
+		"install": "node-scripts setup"
```

Comment:
This is the `install` hook that is triggered when `yarn install` is run and invokes `node-scripts setup`.

Reply:
Should this be done postInstall though? Just thinking if anythign in the setup script woudl depend on a node_module

Reply:
According to this `postinstall` is better since it is run only once -> https://docs.npmjs.com/cli/v6/using-npm/scripts#examples

However then they say in that same place that it's OK to use `make` in `install` :flushed: 

Let's see what happens at CI and I'll decide based on that :shrug: 
Locally `install` seems to be running in my PC and it's in fact depending on other packages to extract the tar.gz files so...

Reply:
It works even after a git clean that deletes de node_modules completely.